mitodl/micromasters
# add review time of 5 days to financial aid email

#### What are the relevant tickets?

#4159 

#### What's this PR do?

attempts to make it clearer to learners that they need to wait at least 5 days after their documents have arrived before they will get their personal price.

#### How should this be manually tested?

1. Make sure your dev environment is setup to send financial aid emails
1. Submit a personal price for a user.  
2. Go to the financial aid review form and find the user's submission (probably under `/pending-docs/` or "Started Application")
3. Click the "Mark Docs Received" button

#### Any background context you want to provide?

A significant portion of our ZenDesk email is from learners checking on the status of their income submission. Hopefully this will cut down on it a bit.
